heartland-retail.gemspec

```
@@ -5,7 +5,7 @@ Gem::Specification.new do |s|
   s.authors           = ['Jay Stotz', 'Derek Stotz']
   s.summary           = 'Heartland Retail API client library'
 
-  s.add_runtime_dependency 'faraday', '~> 1.0'
+  s.add_runtime_dependency 'faraday', '>= 1.0'
```

It's better to force a specific version so that we know that our current revision is working. By introducing a range of versions, if the dependency updates and introduces a breaking change (willingly or not) our gem would also break. Then the correct way to keep the gem updated would be to introduce a cadence for updating the dependency instead of relying on the range.
In this case, seeing how the most current version of faraday is 2.12.2 and it looks like it will work without issues, let's use that:
```suggestion
  s.add_runtime_dependency 'faraday', '2.12.2'
```